contracts/governance/GovernanceSettings.sol

```diff
@@ -3,51 +3,133 @@ pragma solidity 0.8.27;
 
 import {Decimal} from "../common/Decimal.sol";
 import {Governable} from "../model/Governable.sol";
-import {SoftwareUpgradeProposal} from "../proposal/SoftwareUpgradeProposal.sol";
-import {Constants} from "./Constants.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
 
 /// @notice GovernanceSettings is a contract for managing governance settings
-contract GovernanceSettings is Constants {
-    uint256 constant _proposalFee = _proposalBurntFee + _taskHandlingReward + _taskErasingReward;
-    uint256 constant _proposalBurntFee = 50 * 1e18;
-    uint256 constant _taskHandlingReward = 40 * 1e18;
-    uint256 constant _taskErasingReward = 10 * 1e18;
-    uint256 constant _maxOptions = 10;
-    uint256 constant _maxExecutionPeriod = 3 days;
+contract GovernanceSettings is Ownable {
+    uint256 private _proposalFee;
+    uint256 private _proposalBurntFee;
+    uint256 private _taskHandlingReward;
+    uint256 private _taskErasingReward;
+    uint256 private _maxOptions;
+    uint256 private _maxExecutionPeriod;
+
+    // reverted when proposal fee would be under the sum of burnt fee and rewards
+    error ProposalFeeTooLow(uint256 necessaryValue);
+
+    constructor() Ownable(msg.sender) {
+        // Set default values
+        _proposalBurntFee = 50 * 1e18;
+        _taskHandlingReward = 40 * 1e18;
+        _taskErasingReward = 10 * 1e18;
+        _maxOptions = 10;
+        _maxExecutionPeriod = 72 hours;
+        _proposalFee = _proposalBurntFee + _taskHandlingReward + _taskErasingReward;
+    }
 
     /// @notice proposalFee is the fee for a proposal
     /// @return proposal fee
-    function proposalFee() public pure returns (uint256) {
+    function proposalFee() public view returns (uint256) {
         return _proposalFee;
     }
 
     /// @notice proposalBurntFee is the burnt part of fee for a proposal
     /// @return proposal burn fee
-    function proposalBurntFee() public pure returns (uint256) {
+    function proposalBurntFee() public view returns (uint256) {
         return _proposalBurntFee;
     }
 
     /// @notice taskHandlingReward is a reward for handling each task
     /// @return task handling reward
-    function taskHandlingReward() public pure returns (uint256) {
+    function taskHandlingReward() public view returns (uint256) {
         return _taskHandlingReward;
     }
 
     /// @notice taskErasingReward is a reward for erasing each task
     /// @return task erasing reward
-    function taskErasingReward() public pure returns (uint256) {
+    function taskErasingReward() public view returns (uint256) {
         return _taskErasingReward;
     }
 
     /// @notice maxOptions maximum number of options to choose
```

Comment:
`maximum number of options to choose` -> `maximum number of options that can be offered in a single proposal`